FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+[uuid(8d3422bf-66df-5bae-9916-71fd23d5bef6), object, pointer_default(unique)]
+interface ICoreWebView2StagingFindActiveMatchIndexChangedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingFind* sender,
+      [in] IUnknown* args);
+}
+/// Receives `FindMatchCountChanged` events.
+[uuid(cecb8e8f-b6c8-55c3-98b1-68fd1e2b9eea), object, pointer_default(unique)]
+interface ICoreWebView2StagingFindMatchCountChangedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke(
+      [in] ICoreWebView2StagingFind* sender,
+      [in] IUnknown* args);
+}
+
+
+/// Interface providing methods and properties for finding and navigating through text in the web view.
+/// This interface allows for finding text, navigation between matches, and customization of the find UI.
+// MSOWNERS: core (maxwellmyers@microsoft.com)
+[uuid(9c494a0a-c5d8-5fee-b7e6-4926d8d7b391), object, pointer_default(unique)]
+interface ICoreWebView2StagingFind : IUnknown {
+  /// Retrieves the index of the currently active match in the find session. Returns the index of the currently active match, or -1 if there is no active match.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propget] HRESULT ActiveMatchIndex([out, retval] UINT32* value);
+
+
+  /// Gets the total count of matches found in the current document based on the last find sessions criteria. Returns the total count of matches.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propget] HRESULT MatchCount([out, retval] UINT32* value);
+
+
+  /// Gets the `ShouldHighlightAllMatches` property.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propget] HRESULT ShouldHighlightAllMatches([out, retval] BOOL* value);
+
+
+  /// Gets or sets the state of whether all matches are highlighted. Returns TRUE if all matches are highlighted, FALSE otherwise.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propput] HRESULT ShouldHighlightAllMatches([in] BOOL value);
+
+
+  /// Gets the `SuppressDefaultFindDialog` property.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propget] HRESULT SuppressDefaultFindDialog([out, retval] BOOL* value);
+
+
+  /// Checks if a custom user interface is desired by the end developer. Returns TRUE if using a custom UI, FALSE if using the default.
+  // MSOWNERS: core (maxwellmyers@microsoft.com)
+  [propput] HRESULT SuppressDefaultFindDialog([in] BOOL value);
+
+
+
+  /// Adds an event handler for the `ActiveMatchIndexChanged` event.
+  /// Registers an event handler for the ActiveMatchIndexChanged event. This event is raised when the index of the currently active match changes. This can happen when the user navigates to a different match or when the active match is changed programmatically. The parameter is the event handler to be added. Returns a token representing the added event handler. This token can be used to unregister the event handler.
```

Comment:
Add line breaks to keep the width of the lines at a reasonable length, e.g. 80 chars or 120 chars.

Reply:
Similarly with the rest of the long lines in this section.

Reply:
Please fix.